core/services/promreporter/prom_reporter_test.go

```diff
@@ -45,7 +45,7 @@ func newLegacyChainContainer(t *testing.T, db *sqlx.DB) legacyevm.LegacyChainCon
 		RpcBatchSize:             2,
 		KeepFinalizedBlocksDepth: 1000,
 	}
-	lp := logpoller.NewLogPoller(logpoller.NewORM(testutils.FixtureChainID, db, lggr, pgtest.NewQConfig(true)), ethClient, lggr, lpOpts)
```

Comment:
If you want to merge this PR before restoring the behavior, then we need to track all of the cases somehow.